### DISCUSSION TITLE
Transmit data to other destination by reconfiguring logstash.conf  but no logs

### DISCUSSION THREAD

**Discussion by leoylf** [telekom-security/tpotce#1535] [Q&A]
Followed this instruction to transmit data to local files but no log was created in the /data/[type]/log folder.(https://github.com/telekom-security/tpotce/wiki/Reconfigure-logstash.conf.

**Comment by dmille6** [comment]
is your installation a single honeypot or a hive install with multiple sensors going to a hive server?

**Reply by leoylf** [reply]
It's a hive installation with multiple sensors going to the hive server. The attacks were logged into the elasticsearch but not written into the output files on the Hive server.

**Reply by leoylf** [reply]
It seems the attack logs are saved on the sensors but not on the Hive server.

**Reply by dmille6** [reply]
yes, logs (and malicious files) are saved on sensors.. then data is sent to ES instance in hive.
